<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contabilidade</class>
 <widget class="QWidget" name="Contabilidade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contabilidade</string>
  </property>
  <widget class="QLabel" name="labelPesquisa">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>11</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pesquisa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipoPesquisa">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Pesquisa:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTipoPesquisa">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>39</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Pessoa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mensalidade</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Item</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Doação</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditPesquisa">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>39</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPesquisa">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewContabilidade">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>931</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>591</y>
     <width>931</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutAcoes">
    <item>
     <widget class="QPushButton" name="pushButtonCalcular">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular total</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonGerarGrafico">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gerar grafico</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
